presto-benchmark-runner/src/main/java/com/facebook/presto/benchmark/framework/StreamExecutionPhase.java

```diff
@@ -0,0 +1,68 @@
+/*
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package com.facebook.presto.benchmark.framework;
+
+import com.fasterxml.jackson.annotation.JsonCreator;
+import com.fasterxml.jackson.annotation.JsonProperty;
+import com.google.common.collect.ImmutableList;
+
+import java.util.List;
+import java.util.Objects;
+
+import static com.google.common.collect.ImmutableList.toImmutableList;
+
+public class StreamExecutionPhase
+        extends PhaseSpecification
+{
+    private final List<List<String>> streams;
+
+    @JsonCreator
+    public StreamExecutionPhase(
+            @JsonProperty("name") String name,
+            @JsonProperty("executionStrategy") ExecutionStrategy executionStrategy,
```

Comment:
This shouldn't be required as part of the json to construct the object. The `@type` already gives enough information to create the object.